src/ansys/fluent/core/quantity.py

```diff
@@ -0,0 +1,184 @@
+from itertools import permutations
+
+import pint
+from pint import Unit
+
+base_units = pint.UnitRegistry(autoconvert_offset_to_baseunit=True)
```

Comment:
So, now that I've stared at `pint` a bunch more probably it's best to just name this `_ureg`, as they call it usually in the `pint` documentation, and make it a private variable.  I don't think there is a reason to make it public.

Reply:
Replacing ```base_units``` with ```_ureg``` is possible. 

but making it private variable is not possible.

We have tried this - 

![image](https://user-images.githubusercontent.com/106588300/186347120-5a78711e-1749-423f-b7b5-2ebd3112f6cf.png)

and by doing this every time new instance is created with new registry and we can not operate instances of 2 different registries together. 

we get this error - 

![image](https://user-images.githubusercontent.com/106588300/186347526-766b29d2-839c-49ba-a436-383807691ad3.png)

It's mentioned here - 
https://pint.readthedocs.io/en/stable/tutorial.html#tutorial

![image](https://user-images.githubusercontent.com/106588300/186348413-cf9e8112-09db-4e97-85f7-a09c9432467c.png)

Reply:
Renamed ```base_units``` to ```ureg```.